### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by peteroneilljr**
Signed-off-by: Peter ONeill <peteroneilljr@gmail.com>

### CODE REVIEW

**Comment by peteroneilljr** [comment]
@tsandall I've added your suggestions, except the for adding a new diagram, but I have created an issue to track that request. PTAL when you have a chance.

https://github.com/StyraInc/opa-aws-cloudformation-hook/issues/27

File: README.md [hunk 1]
```diff
@@ -1,193 +1,15 @@
-# Cloudformation
+# OPA CloudFormation Hook
```
Comment (anderseknert) [inline README.md:1]: We call this "OPA AWS CloudFormation Hook" in the tutorial, and it's also the name of the repo.. so I'd suggest we go with that here too.

File: README.md [hunk 2]
```diff
@@ -1,193 +1,15 @@
...
 
-* A Template is equivalent to a .tf file - i.e. describing the resources to be provisioned
-* A Stack provisions the resources described in a template
+This repository integrates AWS Cloud Formation (CFN) with OPA using [AWS Cloud Formation Hooks](https://aws.amazon.com/about-aws/whats-new/2022/02/aws-announces-general-availability-aws-cloudformation-hooks/). Use this integration if you want to enforce policies over AWS resources (e.g., EC2 instances, S3 buckets, etc.) provisioned with CFN. For example, using this integration you can:
```
Comment (anderseknert) [inline README.md:3]: > For example, using this integration you can:

You can what? :)

**Review by anderseknert — COMMENTED** [review]
Looks like a good start! I'll follow up with a PR later to complement with some detais on how to use this for development. Left two comments.

**Pull request merged**